thirty-three-across/thirty-three-across-htb.js

```diff
@@ -563,8 +583,9 @@ function ThirtyThreeAcrossHtb(configs) {
             var pixelUrl = [
                 SYNC_ENDPOINT,
                 'id=' + (configs.siteId || DEFAULT_SYNC_ID),
-                'gdpr=' + gdprConsent.gdpr,
-                'gdpr_consent=' + encodeURIComponent(gdprConsent.consentString)
+                'gdpr=' + privacyStats.tcf.gdpr,
+                'gdpr_consent=' + encodeURIComponent(privacyStats.tcf.consentString),
+                'us_privacy=' + encodeURIComponent(privacyStats.ccpa.consentString)
             ].join('&');
```

Comment:
What would the URL look like if it's not GDPR and/or CCPA eligible request? The params would have undefined values for consent string, is that OK?

Reply:
gdpr will be undefined and emptied out in the url param, ccpa will be default to empty string, and the url param will also emptied out. but let me confirm this